lib/chains/cookieChain.js

```diff
@@ -52,6 +52,7 @@ const toString = data => {
 			);
 	}
 };
+const methodToString = (data) => `cookie(${data.cookieName || ''})`;
```

Comment:
There is no need for safeguard (`|| ''` part), as cookie name goes through validation.